.eslintrc.js

```diff
@@ -67,11 +67,7 @@ module.exports = {
 		},
 		{
 			plugins: [ 'mocha' ],
-			files: [
-				'test/e2e/**/*',
-				'packages/magellan-mocha-plugin/test/**/*',
```

Comment:
We migrated the tests to Jest a while ago, no need to run the `mocha` eslint plugin on them.